packages/rum-core/src/performance-monitoring/capture-hard-navigation.js

```diff
@@ -51,35 +54,26 @@ const eventPairs = [
   ['loadEventStart', 'loadEventEnd', 'Fire "load" event']
 ]
 
-function isValidSpan(transaction, span) {
-  const duration = span.duration()
-  return (
-    duration < SPAN_THRESHOLD &&
-    duration > 0 &&
-    span._start <= transaction._end &&
-    span._end <= transaction._end
-  )
-}
-
-function isValidPerformanceTiming(start, end, baseTime = 0) {
+function shouldCreateSpan(start, end, baseTime, transactionEnd) {
   return (
     typeof start === 'number' &&
     typeof end === 'number' &&
+    typeof baseTime === 'number' &&
     start >= baseTime &&
     end > start &&
-    end - start < SPAN_THRESHOLD &&
-    start - baseTime < SPAN_THRESHOLD &&
```

Comment:
this check is basically not useful required. It sometimes even backfires like when one of the `startTime` or `fetchStart` is present in nav timing. In that case the long value would always be more than Threshold (30000) and would not create a span.